src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoServiceSpec.scala

```diff
@@ -54,5 +63,10 @@ class LeonardoServiceSpec extends TestKit(ActorSystem("leonardotest")) with Flat
     }
   }
 
+  "LeonardoService" should "initialize bucket with correct files" in isolatedDbTest {
+
+    leo.initializeBucket("googleProject", "cluster", "bucket")
+  }
+
```

Comment:
Here, I want to test whether or not leo is initializing the bucket (creating it and then adding the objects). Can I call the DAO directly to check whether these things have been added? Or does everything have to go through the service?

Reply:
yup, same as my answer above: you can totally peek into the mock dao to make sure leo called the functions you expected it to